python_modules/dagster/dagster/__init__.py

```diff
@@ -307,6 +312,21 @@
         "1.1.0",
         "Instead, use an input manager or root input manager.",
     ),
+    "FileHandle": (
+        "dagster._core.storage.file_manager",
+        "1.1.0",
+        "It is recommended to handle I/O to a filesystem within an IO manager.",
+    ),
+    "LocalFileHandle": (
+        "dagster._core.storage.file_manager",
+        "1.1.0",
+        "Local file I/O can be handled by the fs_io_manager.",
+    ),
+    "local_file_manager": (
```

Comment:
Assuming we're not targeting a removal date for file manager, we should take out these deprecations.

Reply:
roger